.eslintrc

```diff
@@ -4,24 +4,72 @@
     "plugin:@typescript-eslint/eslint-recommended",
     "plugin:@typescript-eslint/recommended",
     "prettier",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "plugin:import/recommended",
+    "plugin:import/typescript"
+  ],
+  "plugins": [
+    "prettier",
+    "@typescript-eslint",
+    "import"
   ],
-  "plugins": ["prettier", "@typescript-eslint", "simple-import-sort"],
   "parser": "@typescript-eslint/parser",
   "rules": {
-    "id-length": ["warn", { "min": 2, "exceptions": ["_", "t"] }],
-    "prettier/prettier": ["error", {}, { "usePrettierrc": true }],
-    "simple-import-sort/imports": "error",
+    "prettier/prettier": [
+      "error",
+      {},
+      {
+        "usePrettierrc": true
+      }
+    ],
     "@typescript-eslint/ban-ts-ignore": "off",
     "@typescript-eslint/ban-ts-comment": "off",
-    "no-console": ["warn"]
+    "@typescript-eslint/consistent-type-imports": [
+      "error",
+      {
+        "prefer": "type-imports",
+        "fixStyle": "separate-type-imports"
+      }
+    ],
```

Comment:
dev: this is copy-paste from the `@causaly/platform` private repo. The only change is that it throws an error instead of a warning.